appliances/backup_and_restore.md

```diff
@@ -15,6 +15,22 @@ Creating a backup of the appliance will be done using SSH or the console.
 
 3. Save and/or rename `/root/backup.tgz`
 
+# Scheduling Database Backups
+Scheduling a database backup using crontab and pg_dump
+
+1. SSH into the appliance and edit the root user crontab `crontab -e`
+2. Press `i` to insert and add a line with the desired schedule to run the backup job. Examples:
+   ```bash
+     # Every hour on the hour
+     0 * * * * pg_dump -Fc vmdb_production > /tmp/db_dump$(date +\%Y\%m\%dT\%H\%M\%S).dump
+     # Every day at midnight
+     0 0 * * * pg_dump -Fc vmdb_production > /tmp/db_dump$(date +\%Y\%m\%dT\%H\%M\%S).dump
+     # Every week at midnight on Sunday
+     0 0 * * 0 pg_dump -Fc vmdb_production > /tmp/db_dump$(date +\%Y\%m\%dT\%H\%M\%S).dump
```

Comment:
Shoudl we consider using appliance_console_cli for this? I assume it's the `--dump` option.

Reply:
+1 - I was also expecting this PR to be showing a cron calling into application console cli

Reply:
Good point.  I'm fine with doing this for now.  I guess the underlying command could be moved down to a lower level and running through vanilla ruby and not need all of the rails environment.

Reply:
I'll merge for now and we can review the invocation of pg_dump vs. appliance_console_cli along with our existing usage of pg_dumpall and pg_basebackup.  Hopefully, we can specify this information in less places so it's easier to maintain going forward.

Reply:
> underlying command could be moved down to a lower level and running through vanilla ruby and not need all of the rails environment.

To clarify, the appliance_console_cli shouldn't need the Rails env.  However, since it is a gem, the PATH and ENV do need to be updated to find that gem.